x-pack/plugins/lens/public/trigger_actions/open_in_discover_action.ts

```diff
@@ -5,17 +5,23 @@
  * 2.0.
  */
 
-import type { IEmbeddable } from '@kbn/embeddable-plugin/public';
 import { i18n } from '@kbn/i18n';
 import { createAction } from '@kbn/ui-actions-plugin/public';
-import type { DiscoverStart } from '@kbn/discover-plugin/public';
-import type { Embeddable } from '../embeddable';
-import { DOC_TYPE } from '../../common';
+import type { DiscoverSetup } from '@kbn/discover-plugin/public';
```

Comment:
Again, shouldn't this be `DiscoverStart`?

Reply:
Reverted